master/networking/mtu.md

```diff
@@ -3,127 +3,138 @@ title: Configuring MTU
-attached to each workload and the tunnel devices if IP-in-IP or VXLAN is enabled.
+### Big picture
 
-### Selecting MTU size
+Configure the maximum transmission unit (MTU) for your {{site.prodname}} environment.
 
-Since MTU is a global property of the network path between endpoints, the MTU for 
-workloads needs to be set to the minimum MTU of any path that packets may take.
+### Value
 
-If you are using an overlay such as IP-in-IP or VXLAN, the extra overlay header
-used by those protocols reduces the minimum MTU by the size of the header.  IP-in-IP uses
-a 20-byte header, VXLAN uses a 50-byte header.  Hence,
+Optimize network performance for workloads by configuring the MTU in {{site.prodname}} to best suit your underlying network.
 
-- If you use VXLAN anywhere in your pod network, you should select an MTU which is
-  network MTU minus 50.
-- If you do not use VXLAN but you do use IP-in-IP, you should select an MTU which is
-  network MTU minus 20.
-  
-You should set the workload endpoint MTU **and** the tunnel MTUs to the same value.
-This is so that all paths have the same MTU.
+Increasing the MTU can improve performance, and decreasing the MTU when it is too high can resolve packet loss and fragmentation problems.
 
-#### Common MTU sizes
+### Features
 
-| Network MTU | {{site.prodname}} MTU | {{site.prodname}} MTU with IP-in-IP | {{site.prodname}} MTU with VXLAN (IPv4) |
-|-------------|------------|--------------------------|------------------------------|
-| 1500 | 1500 | 1480 | 1450 |
-| 9000 | 9000 | 8980 | 8950 |
-| 1460 (GCE) | 1460 | 1440 | 1410 |
-| 9001 (AWS Jumbo) | 9001 | 8981 | 8951 |
-| 1450 (OpenStack VXLAN) | 1450 | 1430 | 1400 |
+This how-to guide uses the following {{site.prodname}} features:
 
-#### Default MTU sizes
+- **calico-config ConfigMap** file
+- **FelixConfiguration** file
 
-The default MTU for workload interfaces is 1500, this is to match the most
-common network MTU size. The default MTU for the IP-in-IP tunnel device
-is 1440 to match the value needed in GCE.  Similarly, the default for VXLAN
-is 1410.
+### Concepts
 
-#### Using flannel for networking
+#### MTU and {{site.prodname}} defaults
 
-When using flannel for networking, the MTU for the network interfaces
-should match the MTU of the flannel interface.  In the above table the 4th
-column "{{site.prodname}} MTU with VXLAN" is the expected MTU when using flannel
-configured with VXLAN.
+The maximum transmission unit (MTU) setting determines the largest packet size that can be transmitted through your network. MTU is configured on the veth attached to each workload, and tunnel devices (if you enable IP in IP and/or VXLAN).
 
-## MTU configuration
+The default MTU sizes are:
 
-It is the job of the network plugin to create new workload interfaces.  The CNI plugin, which is used by
-Kubernetes, supports configuring the MTU of the workload interface through the CNI configuration file.
+- Workloads (veth), 1440
```

Comment:
I think this isn't quite right.

The workloads do _default_ to 1440, but if using VXLAN you need to adjust them to be 1410.

Reply:
Deleted this section. It is explained later.